js/form.js

```diff
@@ -0,0 +1,108 @@
+const form = document.querySelector('.ad-form');
+const selectedType = form.querySelector('#type');
+const priseField = form.querySelector('#price');
+const priseSlider = form.querySelector('#slider'); //<div> слайдера
+const timeinField = form.querySelector('#timein');
+const timeoutField = form.querySelector('#timeout');
+const selectedRoom = form.querySelector('#room_number');
+const selectedGuest = form.querySelector('#capacity');
+
+
+const pristine = new Pristine(form,{
+  classTo: 'ad-form__element',  // Элемент, на который будут добавляться классы
+  errorTextParent: 'ad-form__element', // Элемент, куда будет выводиться текст с ошибкой
+  errorTextTag: 'p', // Тег, который будет обрамлять текст ошибки
+  errorTextClass: 'prestine__error-text' // Класс для элемента с текстом ошибки
+},
+true);
+
+const minPriselist =
+{
+  'bungalow' : [0,'бунгало'],
+  'flat' : [1000,'квартиру'],
+  'hotel' : [3000,'отель'],
+  'house' : [5000,'дом'],
+  'palace' : [10000,'дворец']
+};
+
+
+const validatePrise=(valuePrise) => {
+  const minPrise = minPriselist[selectedType.value][0];
+  return minPrise <= valuePrise; //&& valuePrise <= 100000
+};
+
+const getPriseErrorMessage=() => {
+  const minPrise = minPriselist[selectedType.value][0];
+  const ruTypeName = minPriselist[selectedType.value][1];
```

Comment:
Здесь можно использовать [деструктуризацию](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment):
```suggestion
  const [minPrise, ruTypeName] = minPriselist[selectedType.value];
```

С одним элементом её тоже можно использовать.